arex-storage-web-api/src/main/java/com/arextest/storage/service/ScheduleReplayingService.java

```diff
@@ -110,11 +119,20 @@ public List<AREXMocker> queryRecordList(ViewRecordRequestType request) {
         }
         result = queryRecordsByProvider(providerName, recordId, mockCategoryTypes);
         if (CollectionUtils.isNotEmpty(result)) {
+          responseType.setSourceProvider(providerName);
           break;
         }
       }
     }
 
+    // find recording scene data
+    if (findRecordingScene) {
+      AREXMocker arexMocker = recordingSceneMockerHandler.findByRecordId(recordId);
+      if (arexMocker != null) {
+        result.add(arexMocker);
+      }
```

Comment:
this if logic can it be merged onto `if (findRecordingScene)` ?

Reply:
optimized